MapboxNavigation/NavigationViewController.swift

```diff
@@ -314,6 +314,11 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         drawerCornerRadius = 0
         progressBar.dock(on: view)
         self.delegate = self
+        
+        if routeController.isSimulationModeEnabled {
```

Comment:
Try moving this to didAppear. It also improve discoverability as it will animate when you're the view is present. Easier to miss the status message if it's already there when you open the screen.
Also, adding a duration here might be useful.